src/dev.rs

```diff
@@ -0,0 +1,343 @@
+//! Tools for developing circuits.
+
+use ff::Field;
+
+use crate::{
+    arithmetic::{FieldExt, Group},
+    plonk::{permutation, Any, Assignment, Circuit, Column, ConstraintSystem, Error},
+    poly::{EvaluationDomain, LagrangeCoeff, Polynomial},
+};
+
+#[derive(Debug)]
+struct Cell(usize, usize);
```

Comment:
Does this need to be introduced in this PR? It's not used anywhere AFAICT. If it's helpful, though, let's introduce it in the `circuit` module instead.

Also, consider doing `(Column, usize)` instead of `(usize, usize)`.

Reply:
It is leftover from my placeholder Permutation collector, which Ying Tong replaced. I'll remove it.